nih-sparc/sparc-app
# Rename 'map' to 'maps'. Fix usage of wrong environment variable.

# Description

Renames `map` to `maps`.  Moves from using the `sparc_api_host` environment variable to  `portal_api` environment variable.


## Type of change

Delete those that don't apply.


# How Has This Been Tested?

1. Navigate to the Find Data page.
2. Click on a dataset that contains images 'Distribution of ICN Neurons in Male and Female 3D Reconstructed Rat Hearts'.
3. Go to Gallery tab.
4. Thumbnail images gathered from the dataset should be visible in a strip with some contextual information.
5. Click on the last thumbnail (the scaffold thumbnail).
6. In a new tab a viewer should open for the scaffold viewer.  The scaffold viewer will show a heart with a beta icon in the top left corner.

# Checklist:
